NEWS.md

```diff
@@ -1,3 +1,8 @@
+## 5.0.0 IN PROGRESS
+ * [FOLSPRINGB-58](https://issues.folio.org/browse/FOLSPRINGB-58) Async FolioExecutionContext helper
+
+Breaking change: Replace calls to `FolioExecutionScopeExecutionContextManager.beginFolioExecutionContext` and  `FolioExecutionScopeExecutionContextManager.endFolioExecutionContext` with `FolioExecutionContextSetter` usage.
```

Comment:
Having AutoClosable wrapper for FolioExecutionScopeExecutionContextManager looks good to me, but at the same time it cannot be used as try-with-resources in many cases. For example in: mod-data-export-worker in org/folio/dew/config/kafka/KafkaConsumerInterceptor.java:34 the context is started in one method, and should be closed in another. For such cases change usage of FolioExecutionScopeExecutionContextManager to FolioExecutionContextSetter looks redundant.